gatorgrouper/utils/run.py

```diff
@@ -52,7 +52,9 @@ def input_interface(
     """ Run conditional logic statment to ran different methods """
     students_reshuffle = group_creation.shuffle_students(students)
     if method == constants.ALGORITHM_ROUND_ROBIN:
-        if (len(students_reshuffle[0])) < 2:  # Note that this only checks the first student
+        if (
+            len(students_reshuffle[0]) < 2
+        ):  # Note that this only checks the first student
```

Comment:
Is this from black? it should look like `if len(students_resuffle[0]) < 2:`

Reply:
Yea it is from black, I changed it and black reformated it back

Reply:
I just checked and black should not reformat that code. The below code, matching exactly the part that matters, is not reformatted.

```
students_reshuffle = [[23], ["t"]]
if len(students_reshuffle[0]) < 2:
    print("tested")

```

Reply:
Ah, it's because of the tailing comment. That should be on the line above the `if`, as should the `pylint: disable` statements scattered throughout this code.

Reply:
I have tried again, black does change it. Is it possible due to the version? I have `black, version 19.3b0`

Reply:
oh, thanks. I'm going to change it.

Reply:
fixed